gulpfile.js

```diff
@@ -29,7 +31,11 @@ function runSpecs(argument) {
     }));
 
   return spec.on('end', function () {
-    process.exit(0);
+    if (IS_LOCAL) {
+      process.exit(0);
```

Comment:
Ah, my fault, the problem is that you should split the cover report sending and exit as the two blocks.